akka-slf4j/src/main/scala/akka/event/slf4j/Slf4jLogger.scala

```diff
@@ -141,3 +142,16 @@ class Slf4jLoggingFilter(settings: ActorSystem.Settings, eventStream: EventStrea
   def isDebugEnabled(logClass: Class[_], logSource: String) =
     (eventStream.logLevel >= DebugLevel) && Logger(logClass, logSource).isDebugEnabled
 }
+
+/** Wraps [[org.slf4j.Marker]] */
+final class Slf4jLogMarker(marker: org.slf4j.Marker) extends LogMarker(name = marker.getName) {
```

Comment:
I see how Java API can get at underlying `org.slf4j.Marker`, but don't see how Scala API gets at it -- there is no `val marker` and it's not a case class, so `marker` is not accessible.